src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverseObject = {};
```

Comment:
Try to make up a more descriptive name, for example, `inversedRobot`. Try not to use the words `object`, `array`, and so on in your variables names